app/code/Magento/InventorySwatchesFrontendUi/Test/Mftf/Test/StorefrontDisplayCorrectMessageWhenChildProductSelectedForConfigProductTest.xml

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontDisplayCorrectMessageWhenChildProductSelectedForConfigProductTest">
+        <annotations>
+            <features value="ConfigurableProduct"/>
+            <stories value="Configurable Product Child Quantity"/>
+            <title value="Display 'Only X left' message when a child product is selected on Configurable product page on the Storefront"/>
+            <description value="Display 'Only X left' message when a child product is selected on Configurable product page on the Storefront"/>
+            <testCaseId value="MC-17951"/>
+            <useCaseId value="MC-17694"/>
+            <severity value="MAJOR"/>
+            <group value="сonfigurable_product"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <magentoCLI command="config:set  cataloginventory/options/stock_threshold_qty 2" stepKey="setQty2"/>
```

Comment:
xample of good mftf fixture https://github.com/magento-epam/magento2ce/blob/2.3-develop/app/code/Magento/Captcha/Test/Mftf/Data/CaptchaConfigData.xml 
than you can use in the test such format

`<magentoCLI command="config:set {{StorefrontEnableAddStoreCodeToUrls.path}} {{StorefrontEnableAddStoreCodeToUrls.value}}" stepKey="addStoreCodeToUrlEnable"/>`